app/views/posts/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<%= render partial: "shared/user", locals: { user: @user } %>
+<% @user.posts.each do |post| %>
+  <div class="post-detailed">
+    <p><%= post.title %></p>
+    <p><%= post.text %></p>
+    <div class="user-postdata">
+      <p>Comments: <%= post.comments_counter %></p>
+      <p>Likes: <%= post.likes_counter %></p>
+    </div>
```

Comment:
- [ ] Please implement a link on the post so that whenever a post is clicked, it takes us to the post's show page 👍. This will improve the UX of your app as user's wont have to manually type in the URL to get to view a post like this: `http://127.0.0.1:3000/users/1/posts/4`

Reply:
The posts are now links!